enhancements/single-node/aos-sno-pair.md

```diff
@@ -0,0 +1,143 @@
+The RHEL-HA cluster performs periodic health checking of Deployments at two levels, frequently checking the number of requested replicas is correct, but also checking the number of active replicas at a lower frequency in order to capture additional soft-failure scenarios where the scheduler is unable to satisfy the replica count.
+
+It is imagined that most Deployments would be scaled between 0 and 1 but the minimum and maximum counts are admin configurable for added flexibility.
+
+Rather than creating a one-size-fits-all 2 node architecture, supportable customer deployments must include a minimum number of [redundancy options](https://drive.google.com/file/d/1aw_78490rTQGsxEFcZytMe8lasZErLMm/view?usp=sharing) based on the constraints of their environment.
+
+#### Network Redundancy
+Bonding or Knet can be configured with redundant network paths in environments where this is possible.  This helps ensure that the RHEL-HA cluster does not attempt recovery until all communication paths between the peers have been exhausted.
+
+Care must be taken to ensure that paths are truly independent and avoid single points of failure (SPoF). Crossover cables are often the simplest way of meeting this requirement and are therefore highly encouraged.
+
+#### Fencing
+In the event of a peer failure, any **workloads requiring singleton semantics** (such as StatefulSets or kubevirt VMs) require the existing node to be put into a safe state (stopped or otherwise known to be not running the workloads) before they can be started on the surviving peer.  The traditional term for this is “**fencing**” and it is an **optional** part of RHEL-HA clusters.
+
+Customer environments without any singleton workloads may not require fencing, but need to be comfortable with the possibility of workloads being active on both peers at the same time.
+
+For customers that do have singleton workloads, the RHEL-HA stack supports a wide variety of BMCs including those conforming to the IPMI and RedFish standards, as well as SSH and libvirt for testing.
+
+It also supports indirect fencing methods like poison-pill which can optionally make use of shared storage for out-of-band signalling and heartbeating where available.
+
+#### Quorum handling
+_The "two node cluster" is a use case that requires special consideration.  With a standard two node cluster, each node with a single vote, there are 2 votes in the cluster. Using the simple majority calculation (50% of the votes + 1)  to calculate quorum, the quorum would be 2.  This means that the both nodes would always have to be alive for the cluster to be quorate and operate._
+<br>Source: man votequorum
+
+For environments in which customers will allow it, Corosync also ships a lightweight quorum arbitrator that can be run locally or in the cloud on any non-cluster node.  In such cases, no special quorum handling is required and careful placement of the arbitrator also ensures that the surviving peer is reachable by its intended clients.
```

Comment:
RHEL-HA

Reply:
We've recently had some conversations about the difference in support lifecycles for RHEL and OCP. It would be good to call that out as a risk in this doc.

Reply:
Aren't RHEL based products generally supported for much longer than OCP?  What specific concerns have been raised?

The RHEL-HA team has a long history of being extremely responsive when bugs are found and proactive when it comes to backports.

Reply:
IIRC, the RHEL lifetime was less than what we've been asked to support for OCP. Either way, the timelines don't always align, which makes extended support challenging sometimes.